src/sentry/api/issue_search.py

```diff
@@ -92,6 +94,27 @@ def convert_status_value(value, projects, user, environments):
     return parsed
 
 
+def convert_category_value(value, projects, user, environments):
```

Comment:
Would be good to get some tests for these in here: https://github.com/getsentry/sentry/blob/284e980df0018f8baee659999268bdd4c7d08255/tests/sentry/api/test_issue_search.py#L218-L227